### PR TITLE
New page: HTMLInputElement.list

### PR DESCRIPTION
**Pull request by estelle**
part of #35544

### CODE REVIEW

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The **`list`** property of the {{domxref("HTMLInputElement")}} interface is the {{htmlelement("datalist")}} element pointed to by the [`list`](/en-US/docs/Web/HTML/Element/input#list) attribute of the element, returning a {{domxref("HTMLDataListElement")}} or `null` if the `list` attribute is not defined or the `list` attribute's value is not associated with any `<datalist>` in the same tree
+
+The `list` attribute identifies a `<datalist>` in the same tree providing predefined options to the user.
+
+> [!NOTE]
+> This is a read-only property. To associate a `<datalist>` with an element, set the value of the `list` attribute with {{domxref("Element.setAttribute", "setAttribute()")}}.
+
+## Value
+
+A {{domxref("HTMLDataListElement")}} or `null`.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/list/index.md:20]:
```suggestion
An {{domxref("HTMLDataListElement")}} or `null`.
```

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<input id="planet" type="text" list="superhero" />
+<datalist id="superhero">
+  <option value="Azarath" />
+  <option value="Krypton" />
+  <option value="Tamaran" />
+  <datalist></datalist>
+</datalist>
+```
+
+You can retrieve or set the id of the `<datalist>` associated with the `<input>` attribute
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/list/index.md:37]:
```suggestion
You can retrieve the `<datalist>` element associated with the `<input>`:
```

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Given the following HTML:
+
+```html
+<label for="planet">Which planet are you from?</label>
+<input id="planet" type="text" list="superhero" />
+<datalist id="superhero">
+  <option value="Azarath" />
+  <option value="Krypton" />
+  <option value="Tamaran" />
+  <datalist></datalist>
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/list/index.md:33]: Extraneous?

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+title: "HTMLInputElement: list property"
+short-title: list
+slug: Web/API/HTMLInputElement/list
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.list
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`list`** property of the {{domxref("HTMLInputElement")}} interface is the {{htmlelement("datalist")}} element pointed to by the [`list`](/en-US/docs/Web/HTML/Element/input#list) attribute of the element, returning a {{domxref("HTMLDataListElement")}} or `null` if the `list` attribute is not defined or the `list` attribute's value is not associated with any `<datalist>` in the same tree
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/list/index.md:11]:
```suggestion
The **`list`** read-only property of the {{domxref("HTMLInputElement")}} interface returns the {{domxref("HTMLDataListElement")}} pointed to by the [`list`](/en-US/docs/Web/HTML/Element/input#list) attribute of the element, or `null` if the `list` attribute is not defined or the `list` attribute's value is not associated with any `<datalist>` in the same tree.
```

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+The `list` attribute identifies a `<datalist>` in the same tree providing predefined options to the user.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/list/index.md:13]: This looks repetitive. Remove?

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+slug: Web/API/HTMLInputElement/list
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.list
+---
+
+{{ApiRef("HTML DOM")}}
+
+The **`list`** read-only property of the {{domxref("HTMLInputElement")}} interface returns the {{domxref("HTMLDataListElement")}} pointed to by the [`list`](/en-US/docs/Web/HTML/Element/input#list) attribute of the element, or `null` if the `list` attribute is not defined or the `list` attribute's value is not associated with any `<datalist>` in the same tree.
+
+The `list` attribute identifies a `<datalist>` in the same tree providing predefined options to the user.
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/list/index.md:13]:
```suggestion
```

File: files/en-us/web/api/htmlinputelement/list/index.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+```html
+<label for="planet">Which planet are you from?</label>
+<input id="planet" type="text" list="superhero" />
+<datalist id="superhero">
+  <option value="Azarath" />
+  <option value="Krypton" />
+  <option value="Tamaran" />
+</datalist>
+</datalist>
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/list/index.md:32]:
```suggestion
</datalist>
```

**Review by Josh-Cena — APPROVED** [review]
LGTM

**Pull request merged**